drivers/hmis/app/views/hmis_client/services/_table.haml

```diff
@@ -0,0 +1,27 @@
+.table-responsive
+  %table.table
+    %thead
+      %tr
+        %th
+        %th Project
+        %th Entry Date
+        %th Service Date Provided
+        %th Service Name
+        %th
+    %tbody
+      - custom_services.each do |service|
+        %tr
+          %td
+            .ds{class: "ds-color-#{service.data_source_id}"}= ds_short_name_for(service.client.id)
```

Comment:
1) @eanders do you think we should have this DS column or no? I added it to the spec for consistency but I could go either way
2) we preload the client just so we can do this `ds_short_name_for` thing. Seems a bit unnecessary to me, since we already know the data source id, but I'm not sure what all its doing.

Reply:
I think the consistency is worth it on this page.

Reply:
Okay. I forgot that that data source pill thing shows the PersonalID and other info when you hover on it, so this makes sense to me now.

Reply:
I think the data source pill would only have a tooltip if we add this:
https://github.com/greenriver/hmis-warehouse/blob/042bd5b6f97fc572ba4c4888b5e013db80ac0785/app/views/clients/_enrollment_table.haml#L51

It doesn't seem like the other tables in the Enrollment Details tab have that, so I didn't add it here. Should I add it? Should the other tables get it as well?

Reply:
I don't think we need the tooltips on the enrollment details page.  Thanks for checking!